installer/conf/omsagent.d/vmware_esxi.conf

```diff
@@ -0,0 +1,24 @@
+<source>
+  type syslog
+  format /^(?<EventTime>\S+)\s{0,4}(?<HostName>\S+)\s{0,4}(?<ProcessName>\S+)\:\s{0,4}(?<SyslogMessage>.*)$/
+  port 1514
```

Comment:
why do you use this port? how can the user configure the rsyslog to forward the VMWare syslog to this port?